app/javascript/css/table.scss

```diff
@@ -79,6 +83,41 @@ table {
   }
 }
 
+table[data-controller="navigateable-table"] {
+  tbody {
+    tr[data-href] {
+      cursor: pointer;
+    }
+  }
+}
+
+table[data-controller="expandable-table"] {
+  tbody {
+    tr[data-target="expandable-table.row"] {
+      &[data-action="click->expandable-table#toggle"] {
```

Comment:
great use of using the existing stimulus data attribute to also control the style